spec/fixtures/redirects.csv

```diff
@@ -1,3 +1,3 @@
 http://crossroads.net/*,https://www.crossroads.net/:splat,301!,master
-http://${env:CRDS_APP_DOMAIN}/*,https://${env:CRDS_APP_DOMAIN}/:splat,301!
+http://int.crossroads.net/*,https://int.crossroads.net/:splat,301!
```

Comment:
Does this need to be changed for prod?

Reply:
No. this test should only run against one environment due to how it's made. It can't be made dynamic as far as I can tell

Reply:
In fact, it being dynamic is what broke it